CMakeLists.txt

```diff
@@ -115,6 +115,13 @@ else()
   set(DESKTOP OFF)
 endif()
 
+if(DESKTOP AND NOT MSVC AND NOT APPLE)
+  # Linux-specific option.
+  add_definitions(-D_GLIBCXX_USE_CXX11_ABI=0)
```

Comment:
Possible to collect this setting in a variable and use that across the board?
Will keep it more maintainable for future edits.

Reply:
Assuming you mean the c++11 option, I have this in my follow up PR to support both, is that ok?

Reply:
Sounds good. Yeah, I see the hardcoded string being duplicated and was wondering if we can use a variable instead. 
Will wait on the next PR.